src/OpenTelemetry/Trace/ExceptionProcessor.cs

```diff
@@ -73,8 +115,65 @@ public override void OnEnd(Activity activity)
 
             if (snapshot != pointers)
             {
+                var exceptionPointers = Marshal.PtrToStructure<EXCEPTION_POINTERS>(pointers);
+                var exceptionRecord = Marshal.PtrToStructure<EXCEPTION_RECORD>(exceptionPointers.ExceptionRecord);
+                activity.SetTag("exceptionRecord.ExceptionCode", exceptionRecord.ExceptionCode);
+                activity.SetTag("exceptionRecord.ExceptionFlags", exceptionRecord.ExceptionFlags);
+
+                switch (exceptionRecord.ExceptionCode)
+                {
+                case EXCEPTION_CODE.EXCEPTION_COMPLUS:
```

Comment:
Need to dig into https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/coreclr/vm/clrex.cpp#L653.

Reply:
And https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/coreclr/inc/ex.h#L32.